packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1960,7 +1970,7 @@ export class ContainerRuntime extends EventEmitter
             case ContainerMessageType.ChunkedOp:
                 throw new Error(`chunkedOp not expected here`);
             case ContainerMessageType.BlobAttach:
-                this.submit(type, content, localOpMetadata);
+                this.submit(type, content, undefined, localOpMetadata);
```

Comment:
See comment above - I do not think this we should conflate op metadata and local metadata